Resources/Prototypes/_RMC14/Entities/Clothing/Shoes/misc.yml

```diff
@@ -8,6 +8,22 @@
     sprite: _RMC14/Objects/Clothing/Shoes/laceup.rsi
   - type: Clothing
     sprite: _RMC14/Objects/Clothing/Shoes/laceup.rsi
+  - type: ContainerContainer
+    containers:
+      item: !type:ContainerSlot
+  - type: ItemSlots
+    slots:
+      item:
+        name: rmc-gun
+        insertSound: /Audio/_RMC14/Weapons/Guns/Reload/m42a2.ogg
+        ejectSound: /Audio/_RMC14/Weapons/Guns/Reload/m42a2.ogg
+        whitelist:
+          tags:
+          - Holdout
+  - type: Matchbox
```

Comment:
Is this intended? Is this to allow igniting matches with the shoes?

Reply:
That's just leftover from when I copied the section from the Marine Boots. It is possible to use a Shoe's Sole to light a match, so I don't think its malignant at all.